test/InlinePredictionTest.cs

```diff
@@ -395,9 +395,9 @@ internal static List<PredictionResult> MockedPredictInput(Ast ast, Token[] token
             return new List<PredictionResult>
             {
                 (PredictionResult)ctor.Invoke(
-                    new object[] { predictorId_1, "TestPredictor", suggestions_1 }),
+                    new object[] { predictorId_1, "TestPredictor", (uint)56, suggestions_1 }),
```

Comment:
should `56` be a const defined somewhere?  It's used in multiple places

Reply:
Yep, will do.

Reply:
Feedback was addressed. Please take another look when you have time, thanks!